### ISSUE TITLE
[Enhancement] Add format > nmeta conversion

### ISSUE THREAD

**Issue by teamxecuter** [jakcron/NNTools#55]
Do you think you could add nacp > nmeta, cnmt > nmeta and npdm > nmeta conversion to nstool? Right now it only outputs the information in a terminal, which while it is fine for just trying to recreate the nmeta, having a way for it to output as nmeta would be much easier.

**Comment by jakcron** [comment]
I mean sure it can be added. But for what purpose? 

`.nmeta` file seems closely related to producing an Application NSP. Are you looking for a tool to unpack a NSP to SwitchSDK project that "could" be rebuilt?

**Comment by teamxecuter** [comment]
Yeah, a tool like that would be better than what I'm trying to do.

**Comment by jakcron** [comment]
I'll consider it :)

**Comment by adamboy7** [comment]
I second this feature. I'm currently trying to edit a cnmt file I dumped to try and enable recording in a game that has the share button disabled. If I could just exctract the NACP file, hex edit it, then just repack everything without having to use an SDK, that would be amazing.
